Cloudinary/Utils/CLDVar.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  CLDVar.swift
+//  Cloudinary
+//
+//  Created by Sergey Glushchenko on 8/28/17.
+//  Copyright © 2017 Cloudinary. All rights reserved.
+//
+
+import UIKit
+
+open class CLDVar: NSObject {
+    public var variable: String = ""
+    public var value: String?
+    
+    public init(_ variable: String) {
+        self.variable = variable
+    }
+    
+    public init(_ variable: String, _ value: String) {
+        self.variable = variable
+        self.value = value
+    }
+    
+    @objc(initWithVariable:andInteger:)
+    public init(_ variable: String, _ integer: Int) {
+        self.variable = variable
+        self.value = String(integer)
+    }
+    
+    @objc(initWithVariable:andStrings:)
+    public init(_ variable: String, _ strings: [String]) {
+        self.variable = variable
+        self.value = "!\(strings.joined(separator: ":"))!"
+    }
+    
+    open override var description: String {
+        if let value = self.value {
+            return "$\(self.variable)_\(value)"
+        }
+        else {
+            return "$\(self.variable)"
+        }
+    }
+    
+    open func mul(_ integer: Int) -> CLDVar {
+        let op: String = CLDArithmetic.multiply.rawValue
+        self.value = "\(op)_\(integer)"
```

Comment:
Looks like every operation _erases_ the previous value making chaining impossible.

For example, what is the output of:

```
CLDVar("foo").mul(10).add("$bar")
```